### PR TITLE
Update JS vendor libs

### LINKED ISSUE #3784: Update vendor JS libraries

### PR DESCRIPTION
**Pull request by theFeu**
fixes #3784

### CODE REVIEW

**Comment by theFeu** [comment]
@lippserd
Any more changes that need to be done here? :)

File: public/js/icinga/loader.js [hunk 1]
```diff
@@ -135,14 +135,15 @@
...
+                processData: ! isFormData,
+                complete: this.onComplete
             });
 
             req.$target = $target;
             req.url = url;
             req.done(this.onResponse);
             req.fail(this.onFailure);
-            req.complete(this.onComplete);
+            //req.complete(this.onComplete);
```
Comment (lippserd) [inline public/js/icinga/loader.js]: This line is free to remove.

**Comment by theFeu** [comment]
> Are there no updates for Tipsy[...]?

No updates in the tipsy repo in the past 7 years: https://github.com/jaz303/tipsy

**Comment by theFeu** [comment]
> Are there no updates for [...] Sparkline?

We're on v2.1.2 - which is the highest according to: https://omnipotent.net/jquery.sparkline/#s-download

**Comment by nilmerg** [comment]
Just stole this, will apply the changes on my own.

**Comment by theFeu** [comment]
Cheers!

**Pull request merged**